### ISSUE TITLE
Tour dialog should be closed when user touch/click outside it

### ISSUE THREAD

**Issue by nashtechdev01** [andrewnicols/flexitour#1]
For accessibility purposes, tour dialog should be closed when user touch/click outside itself.
Ideally, in my opinion, it should similar to bootstrap's modal.

**Comment by nashtechdev01** [comment]
Hi @andrewnicols ,

As discussed on https://tracker.moodle.org/browse/MDL-60207, there are some suggests remove the user tour part of this change.

Could you please review again and confirm because it already in  0.12.0 release?

Many thanks.

**Comment by andrewnicols** [comment]
This caused a regression - it is no longer possible to have guided interactions with the background. Will be reverting part of this patch.
